### PR TITLE
Fix mods arr traversing.

### PR DESCRIPTION
**Pull request by Yeti-or**

### CODE REVIEW

**Comment by blond** [comment]
Could you add tests for this case?

**Comment by Yeti-or** [comment]
I can, but it would not increase coverage. It is just a typo. Are you  sure we need a redundant test?

**Comment by Yeti-or** [comment]
@blond add test case, but not sure about it name

File: spec/normalize.spec.js [hunk 1]
```diff
@@ -62,6 +62,18 @@ describe('normalize', function () {
             ]);
         });
 
+        it('must support arrays', function () {
+            var decl = { name: 'block', mods: [
+                { name: 'mod', vals: [{ name: 'val' }] },
+                { name: 'mod', vals: [{ name: 'val' }] }
```
Comment (blond) [inline spec/normalize.spec.js]: Переименуй на `mod-1` и `mod-2`, чтобы не путать с тестом про "не должно быть повторений в результате".

File: spec/normalize.spec.js [hunk 2]
```diff
@@ -62,6 +62,18 @@ describe('normalize', function () {
             ]);
         });
 
+        it('must support arrays', function () {
```
Comment (blond) [inline spec/normalize.spec.js]: must support several items

**Pull request merged**